src/serlo-editor/editor-ui/plugin-toolbar/text-controls/utils/blank.ts

```diff
@@ -26,10 +26,17 @@ export function getBlankElement(editor: SlateEditor): Blank | undefined {
 
 export function toggleBlank(editor: SlateEditor) {
   if (isBlankActive(editor)) {
+    let text = undefined
     Transforms.removeNodes(editor, {
-      match: (n) => Element.isElement(n) && n.type === 'blank',
+      match: (node) => {
+        const isHit = Element.isElement(node) && node.type === 'blank'
+        if (isHit) text = node
+        return isHit
+      },
     })
-    // TODO: Blank and also its content disappear. Instead blank content should be added back as text element (also not working in toggleMath)
+    if (text) {
+      Transforms.insertNodes(editor, { text: (text as Blank).correctAnswer })
+    }
```

Comment:
Nice! That solves the case when the selection is collapsed. However, does not work when you select something. But better than before, so keeping it. :)

Reply:
When I select something (or everything) inside a `Blank` it works as expected. 
I assume you mean the case when the Blank and something else is selected?

Reply:
Ah sorry, should have explained that in more detail. I meant selecting a blank and other elements. We can find a solution in the next iteration.